src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -2046,3 +2044,200 @@ def get_custom_locations_oid(cmd, cl_oid):
         log_string += "Unable to enable the 'custom-locations' feature. " + str(e)
         logger.warning(log_string)
         return ""
+
+
+def troubleshoot(cmd, client, resource_group_name, cluster_name, kube_config=None, kube_context=None, no_wait=False, tags=None):
+
+    try:
+
+        # C:\Diagnoser\azure-cli-extensions\src\connectedk8s\azext_connectedk8s\custom.py
+        logger.warning("Diagnoser running. This may take a while ...\n")
+        absolute_path=os.path.abspath(os.path.dirname(__file__))
```

Comment:
what does this path point to? is this the current directory of the user?

Reply:
This path points to the customs.py and using this we can reach the troubleshoot_diagnoser_job.yaml without getting dependent on any other things.